examples/extension/src/background.js

```diff
@@ -7,7 +7,7 @@ env.allowLocalModels = false;
 
 // Due to a bug in onnxruntime-web, we must disable multithreading for now.
 // See https://github.com/microsoft/onnxruntime/issues/14445 for more information.
-env.backends.onnx.wasm.numThreads = 1;
+//env.backends.onnx.wasm.numThreads = 1;
```

Comment:
onnxruntime-web should have always deal with numThreads correctly. (ie. the default value should not fail its function) 

explicitly setting numThreads to 1 is no longer necessary.

Need help verification whether this is true.